### PR TITLE
refactor(log) aliyun use the same interface name

### PR DESCRIPTION
**Pull request by Germiniku**
#### Description (what this PR does / why we need it):

`contrib log aliyun` interface use the same interface name `Logger`.

### CODE REVIEW

File: contrib/log/aliyun/aliyun.go [hunk 1]
```diff
@@ -157,10 +158,8 @@ func toString(v interface{}) string {
 		key = strconv.FormatInt(v, 10)
 	case uint64:
 		key = strconv.FormatUint(v, 10)
-	case string:
-		key = v
```
Comment (tonybase) [inline contrib/log/aliyun/aliyun.go:161]: Why did you delete this?

Comment (Germiniku) [reply contrib/log/aliyun/aliyun.go:161]: type `string` implements fmt.Stringer interface.

Comment (tonybase) [reply contrib/log/aliyun/aliyun.go:161]: I tested, type `string` does not implement the `fmt.Stringer` interface.

**Review by tonybase — APPROVED** [review]
